[2024-01-31T12:26:47.026+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-05T00:00:00+00:00 [queued]>
[2024-01-31T12:26:47.244+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-05T00:00:00+00:00 [queued]>
[2024-01-31T12:26:47.245+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-01-31T12:26:48.057+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello_task> on 2024-01-05 00:00:00+00:00
[2024-01-31T12:26:48.208+0000] {standard_task_runner.py:60} INFO - Started process 94 to run task
[2024-01-31T12:26:48.395+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hola_mundo', 'print_hello_task', 'scheduled__2024-01-05T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/prueba.py', '--cfg-path', '/tmp/tmpk08dvtif']
[2024-01-31T12:26:48.432+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask print_hello_task
[2024-01-31T12:26:50.071+0000] {task_command.py:423} INFO - Running <TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-05T00:00:00+00:00 [running]> on host 60382a02b079
[2024-01-31T12:26:51.868+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='usuario' AIRFLOW_CTX_DAG_ID='hola_mundo' AIRFLOW_CTX_TASK_ID='print_hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-05T00:00:00+00:00'
[2024-01-31T12:26:51.961+0000] {logging_mixin.py:188} INFO - Hola Mundo!
[2024-01-31T12:26:51.962+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T12:26:52.018+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hola_mundo, task_id=print_hello_task, execution_date=20240105T000000, start_date=20240131T122647, end_date=20240131T122652
[2024-01-31T12:26:52.303+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T12:26:52.698+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
